middlewares/stripPrefixRegex.go

```diff
@@ -39,7 +39,9 @@ func (s *StripPrefixRegex) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		r.URL.Path = r.URL.Path[len(prefix.Path):]
+		newPath := r.URL.Path[len(prefix.Path):]
+		log.Debugf("Striping Path Prefix Regex %s from %s = %s", prefix.Path, r.URL.Path, newPath)
```

Comment:
Could you remove this Debug log? This is way too verbose to keep it in production code :)